Dockerfile

```diff
@@ -0,0 +1,11 @@
+from python:3.9
```

Comment:
Would it be possible to bump this to Python 3.10 or 3.11? The `idspy-dictionaries` dependency recommends that we use at least 3.10, and 3.10 is the default on Ubuntu 22.04. 3.11 doesn't offer too many benefits over 3.10, but apparently it might run a little faster, which is nice.

Unfortunately 3.12 isn't an option, as they removed the `distutils` library and `pygacode` needs it to build (something I'm hoping to work on in the near future!).

Reply:
Done